### ISSUE TITLE
Add graph visualization

### ISSUE THREAD

**Issue by FranklinChen** [well-typed/visualize-cbn#2]
An option to create a graph visualization would be tremendously useful.

**Comment by edsko** [comment]
Absolutely, agree completely! The code is already structured to make multiple visualization backends possible (currently ANSI, string, and HTML/JavaScript). PRs welcome :)

**Comment by edsko** [comment]
So #11 does add graph visualization, but it generates an evaluation graph (which is linear, since the evaluation is deterministic). Leaving this issue open because it would still be very nice to have a visualization where the _term_ itself is shown as a graph (rather than using named heap pointers).
